main/imagemagick/APKBUILD

```diff
@@ -25,10 +25,6 @@ build() {
 	sed -i -e \
 		's:DOCUMENTATION_PATH="${DATA_DIR}/doc/${DOCUMENTATION_RELATIVE_PATH}":DOCUMENTATION_PATH="/usr/share/doc/imagemagick":g' \
 		configure
-	local _openmp=
-	case "$CARCH" in
-	s390x) _openmp="--disable-openmp"
-	esac
```

Comment:
@TBK you didn't comment on why this was removed. s390x builder hangs in test suite unless `--disable-openmp` is specified.

CC: @tmh1999

Reply:
I can't recall the reason. Must have been a mistake.

I have created a PR to upgrade imagemagick and rectify the mistake - https://github.com/alpinelinux/aports/pull/3129